README.md

```diff
@@ -8,8 +8,6 @@
 
 Dispatch API client for Python
 
-TODO(cookiecutter): Improve the README file
-
```

Comment:
Not sure if removing this one is really fair, the README is not really improved :P

Reply:
I'd argue that it's better than when it contained a note that it needs to be improved :P

Reply:
<p>I don't agree, it is worse because now it won't remind you that it needs improvement 
<img src="https://github.com/frequenz-floss/frequenz-client-dispatch-python/assets/1031485/31c068bd-0b07-4b94-b858-98cdc499c1c3" alt="think" width="32" />
</p>

Reply:
Every time you see it it's pretty clear it needs improvement :P